src/Microsoft.AspNetCore.Authentication.JwtBearer/Events/JwtBearerChallengeContext.cs

```diff
@@ -40,5 +42,15 @@ public JwtBearerChallengeContext(HttpContext context, AuthenticationScheme schem
         /// WWW-Authenticate header. This property is always null unless explicitly set.
         /// </summary>
         public string ErrorUri { get; set; }
+
+        /// <summary>
+        /// If true, will skip any default logic for this event.
+        /// </summary>
+        public bool Skipped { get; private set; }
```

Comment:
This could still live in BaseContext, but since none of the other events check for skip, doesn't really make sense to move it there now.  In the future we could move this concept to be something that all events support if we wanted.